app/views/shared/_ajax_modal.html.erb

```diff
@@ -1,5 +1,5 @@
-<div id="ajax-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="modal menu" aria-hidden="true">
```

Comment:
Not sure if there is any concern that somebody has "patched" this a11y issue downstream by adding elements w/ those IDs in their application.  Latest blacklight 7 does not use `aria-labelledby` on the modal element FWIW.

I think it may be more likely that a downstream application had an unrelated element w/ the ID `menu` that may have caused all kinds of other issues w/ this attribute.